resources/js/forms/PlotForm.vue

```diff
@@ -316,7 +316,7 @@
           if (!e.response || e.response.status !== 422) {
             this.$notify({
               type: 'error',
-              text: 'Unable to process the request. Please try refreshing the page.',
+              text: this.form.errors,
```

Comment:
this.form.errors is not a string so this would fail. Since the status !== 422 then it actually be empty.

```suggestion
              text: 'Unable to process the request. Please try refreshing the page.',
```